src/data/markdown/docs/03 cloud/05 Project and Team Management/05 SAML SSO.md

```diff
@@ -5,30 +5,36 @@ excerpt: 'Guide on setting up a SAML SSO Identity Provider with k6 Cloud'
+- [Auth0](https://auth0.com/) is tested, but not supported, because it can't map [custom attributes](https://community.auth0.com/t/adding-custom-saml-attribute-when-auth0-is-idp/45691) in assertions.
+- Since we support the open The  [SAML 2.0](https://en.wikipedia.org/wiki/SAML_2.0) standard, you can connect k6 Cloud to practically any SAML 2.0 identity provider, not just the ones documented here.
 
-Learn more about [Security Assertion Markup Language](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) and [SAML 2.0](https://en.wikipedia.org/wiki/SAML_2.0) on Wikipedia.
 
-## IdP and SP setup and user provisioning
+## General setup
 
-To setup SAML SSO for your organization, some requirements should be satisfied and some steps should be taken on your IdP and on k6 Cloud. Assuming you already have admin access to your IdP and the k6 Cloud, this is the checklist to enable SAML SSO for your organization on k6 Cloud:
+To set up SAML SSO for your organization, you need admin access to your k6 Organization and to the admin panel of the IdP you are using. This is currently a somewhat manual process that involves getting in touch with the k6 customer success team. These are the general steps:
 
-1. Install or setup the SAML SSO application on your IdP.
-2. Provide the "IdP Metadata URL" to the customer success team.
-3. Update the application on the IdP with the provided token from the customer success team.
-4. Optionally request to enforce SAML SSO on all existing and new users. This effectively disables username/password authentication for all users, even the owner.
-5. Assign users to the IdP application.
-6. Optionally choose default projects for newly provisioned users to land in on k6 Cloud.
-7. Users can now be provisioned to k6 Cloud web app once they have access to and click on the application icon on their IdP dashboard.
-8. Once the user is provisioned (created and joined to the correct organization/project on k6 Cloud), they can use both IdP- and SP-initiated to sign in. This means that in addition to the clicking on the k6 Cloud application on their IdP dashboard, the user can also use SAML SSO button available on the k6 Cloud web app.
+1. Choose or create the `k6 Cloud` SAML SSO application on your IdP of choice (such as Okta or Azure). You will want to organize your users into _Groups_ on the IdP side. [See below](#idp-specific-setups) for the SAMLSSO urls and setup details needed.
+2. You need to find the **IdP Metadata URL** for your SAML SSO application. This should be forwarded to the k6 customer success team.
+    - In Okta, go to the "Sign on" tab for your IdP application. Use `SAML Signing Certificates->Actions->View IdP metadata` and copy the URL of the page that opens.
+    - In Azure, find the "Single sign-on" menu for your IdP application. Copy the `SAML Certificates->App Federation Metadata Url` field.
```

Comment:
While I agree this does depend on a third-party UX that can change (which is the problem of using images), going by my own experience, it would nevertheless have been very useful to have this kind of brief summary available.